shared/tracker/header.render.desktop.js

```diff
@@ -16,48 +16,52 @@ export default class HeaderRender extends Component {
   }
 
   render () {
-    let headerStyle = (this.props.currentlyFollowing && !this.props.changed) ? styles.headerSuccess : styles.headerNormal
-    let headerTextStyle = styles.headerTextNormal
+    let headerStyle = this.props.currentlyFollowing ? styleHeaderFollowing : styleHeaderNotFollowing
```

Comment:
i moved the styles out into their own objects to help with linting and changed headerSuccess to headerFollowing / notFollowing. Echoing the comment from the top, i think we should talk more about the state of the tracker window vs it being action based